subsys/nrf_rpc/nrf_rpc_uart.c

```diff
@@ -211,7 +211,6 @@ static int send(const struct nrf_rpc_tr *transport, const uint8_t *data, size_t
 {
 	uint8_t crc[2];
 	LOG_HEXDUMP_DBG(data, length, "Sending frame");
-	LOG_HEXDUMP_DBG(crc, CRC_SIZE, "frame crc");
```

Comment:
Why was it removed?

Reply:
Because it was incorrect (it printed `crc` before it was calculated). Also, we print the entire frame so we can always calculate the actual CRC if needed and it's worth reducing the amount of logs.